starter-code/server.js

```diff
@@ -146,7 +147,7 @@ app.listen(PORT, function() {
 ////////////////////////////////////////
 function loadArticles() {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
-  // Put your response here...
+  // Put your response here...  This corresponds to #'s 3 and 4.  This is the Read part of CRUD.
```

Comment:
which method is calling loadArticles. also this is a tricky question but Read is not the CRUD operation in this case